Model/UMEPositionMedal.as

```diff
@@ -0,0 +1,158 @@
+// Ultimate Medals Extended - General Medal representing a position entry
+
+#if DEPENDENCY_ULTIMATEMEDALSEXTENDED
+
+class PositionMedal : UltimateMedalsExtended::IMedal {
+
+    int position;
+    PositionData@ positionData;
+    int offset;
+
+    PositionMedal(int position, PositionData@ positionData, int offset = 0) {
+        this.position = position;
+        @this.positionData = positionData;
+        this.offset = offset;
+    }
+
+    UltimateMedalsExtended::Config GetConfig() override {
+        UltimateMedalsExtended::Config c;
+        c.defaultName = "Top " + this.position;
+
+        if(exportIcon && exportIconColor) {
+            c.icon = positionData.GetColorIcon();
+        } else if (exportIcon) {
+            c.icon = greyColor1 + positionData.icon;
+        } else if (exportIconColor) {
+            c.icon = positionData.color + Icons::Kenney::PodiumAlt;
+        } else {
+            c.icon = '\\$444' + Icons::Kenney::PodiumAlt;;
```

Comment:
444 is changed locally to reuse greyColor1, I'll do a final rebase/force push before merging that